### PR TITLE
Bump version to 2.x for numeraljs

### PR DESCRIPTION
**Pull request by ghost**
2.x has a breaking change, introducing the `.register` function it looks like, so maybe a bump in major version is required?

### CODE REVIEW

**Comment by ghost** [comment]
Actually 2.x has more changes than I realised around the loading of languages/locales, so might require an additional change for how to load them.

https://github.com/adamwdraper/Numeral-js/blob/master/README.md#changelog

**Comment by xomaczar** [comment]
There should not be any changes required...

**Comment by ghost** [comment]
@xomaczar Isn't loading the locales different now? The current version loads a `languages.js` which I think has been changed to `locales.js`?

**Comment by ghost** [comment]
Specifically here: https://github.com/josemarluedke/ember-cli-numeral/blob/master/index.js#L21

**Comment by xomaczar** [comment]
Sure - maybe then (never used the feature before until today). I just upgraded my own fork to the latest numeral and using register()/locale() work out of the box. The e-c-n README.MD need to be updated to reflect new changes.

**Comment by xomaczar** [comment]
I see your last commit included all locales - can we allow users to provide option to include specific locales set

**Comment by xomaczar** [comment]
Instead of all of them

**Comment by ghost** [comment]
The original behaviour of the addon was to load all locales, but I agree, we should change the config options to allow for specific locales.

**Comment by ghost** [comment]
Note the `yarn.lock` has changed alot because I think I'm using the latest version of yarn? I just did a `yarn upgrade numeral`.

**Comment by ghost** [comment]
@xomaczar I'm not sure if it's completely working just yet, still testing.

**Comment by xomaczar** [comment]
@Soliah haven't tested myself is it working?

File: index.js [hunk 1]
```diff
@@ -17,8 +19,14 @@ module.exports = {
 
         var filesToImport = [numeralImport];
 
-        if (this.includeLanguages) {
-          filesToImport.push('languages.js');
+        if (this.includedLanguages) {
```
Comment (josemarluedke) [inline index.js]: s/includedLanguages/includeLanguages

File: package.json [hunk 2]
```diff
@@ -17,11 +17,11 @@
   "homepage": "https://github.com/josemarluedke/ember-cli-numeral",
   "dependencies": {
     "ember-cli-node-assets": "^0.1.4",
-    "numeral": "^1.5.3"
+    "numeral": "^2.0"
```
Comment (xomaczar) [inline package.json:20]: :s/2.0/2.0.4 -for consistency

File: index.js (hunk 1 above)
Comment (xomaczar) [reply index.js]: We need tests to catch stuff like that

**Comment by ghost** [comment]
Not working. 

The locales have changed quite a bit and I think we'll need to shim them too. I'm not familiar enough with AMD modules though.

**Comment by ghost** [comment]
Really not sure how to go about getting the locales loaded, especially individually. Any help would be appreciated.

**Pull request closed without merging**